cmake/Projects.cmake

```diff
@@ -143,28 +143,36 @@ foreach(project ${LY_PROJECTS})
     ly_generate_project_build_path_setreg(${full_directory_path})
     add_project_json_external_subdirectories(${full_directory_path})
 
+    # Get project name
+    o3de_read_json_key(project_name ${full_directory_path}/project.json "project_name")
```

Comment:
Can you update the [launcher_generator.cmake](https://github.com/o3de/o3de/blob/development/Code/LauncherUnified/launcher_generator.cmake#L14-L35) to just use `o3de_read_json_key` as well?